helm/minio/templates/podmonitor.yaml

```diff
@@ -1,36 +1,38 @@
-{{- if .Values.metrics.serviceMonitor.enabled }}
+{{- if .Values.metrics.podMonitor.enabled }}
 apiVersion: monitoring.coreos.com/v1
-kind: ServiceMonitor
+kind: PodMonitor
```

Comment:
I am still not sure why this should be a PodMonitor instead of a serviceMonitor - we are basically Monitoring a single deployment here so a ServiceMonitor is correct, prometheus in MinIO provides same information on all nodes. 

There is no pod specific information its always cluster level.

Reply:
When you install chart, there are 3 of services created:
- minio
- minio-console
- minio-svc
- 
ServiceMonitor point to all of them. So, in target you wil see one pod three times (with the wrong port once).

![изображение](https://user-images.githubusercontent.com/18194732/133857713-3682d497-8c02-4780-8e59-b334dc39b9b4.png)

There can be another solution, labels in one service should be different

Reply:
> When you install chart, there are 3 of services created:
> 
> * minio
> * minio-console
> * minio-svc
> 
> ServiceMonitor point to all of them. So, in target you wil see one pod three times (with the wrong port once).
> 
> ![изображение](https://user-images.githubusercontent.com/18194732/133857713-3682d497-8c02-4780-8e59-b334dc39b9b4.png)
> 
> There can be another solution, labels in one service should be different

That is correct solution here @dyvol777

Reply:
What is correct solution, PodMonitor or changing labels in service?

Reply:
Changing the labels @dyvol777

Reply:
Is it ok now? @harshavardhana